### DISCUSSION TITLE
Get-Command: The term 'oh-my-posh' is not recognized as a name of a cmdlet, function, script file, or executable program.

### DISCUSSION THREAD

**Discussion by ClementNerma** [JanDeDobbeleer/oh-my-posh#3209] [Troubleshoot]
### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### What happened?

I installed Oh-My-Posh using the official documentation, through the Windows Store.

But when I run `oh-my-posh` I get the following error:

```
Get-Command: The term 'oh-my-posh' is not recognized as a name of a cmdlet, function, script file, or executable program.
```

Same goes for any command like `Set-PoshPrompt` etc.

I tried installing through the Store, through Winget, through the manual command provided in the docs, nothing works.

I restarted my terminal each time but nothing seems to work.

I'm using Powershell v7.1.

### Theme

N/A

### What OS are you seeing the problem on?

Windows

### Which shell are you using?

powershell

### Log output

```Shell
N/A
```

**Comment by JanDeDobbeleer** [comment]
@ClementNerma the installers add everything to you PATH, did you try restarting your computer? Depending on the Windows version there's different behavior unfortunately.
